snapcraft/internal/sources.py

```diff
@@ -27,7 +27,7 @@
     directory tree or a tarball or a revision control repository
     ('git:...').
 
-  - source-type: git, bzr, hg, tar or zip
+  - source-type: git, bzr, hg, tar, zip, or subversion
```

Comment:
I know it's stupid, but can we have svn next to the other version control systems and leave tar and zip at the end?